deps/3rdparty/libdispatch/src/queue.c

```diff
@@ -675,6 +696,9 @@ _dispatch_queue_dispose(dispatch_queue_t dq)
 	if (slowpath(dq->dq_items_tail)) {
 		DISPATCH_CRASH("Release of a queue while items are enqueued");
 	}
+	if (slowpath(dq->dq_items_tail)) {
+		DISPATCH_CRASH("Release of a queue while items are enqueued");
```

Comment:
it looks like this code's just been copied

Reply:
Woops... I was copying from a backup of the formatted file and copied the wrong lines.